go/desktop-pause.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Pause and Resume Docker Desktop
+description: Instructions on how to pause and resume Docker Desktop
+keywords: pause, resume, docker desktop, mac, windows
+redirect_to: /desktop/mac/#pauseresume
```

Comment:
Ok, this may be tricky, because `/go/desktop-pause/` will now always redirect to the Docker Desktop _for Mac_ manual.

Given that the feature is the same on Mac and Windows (besides a different screenshot of the menu), should we have a top-level (`https://docs.docker.com/desktop/` or `https://docs.docker.com/desktop/<some page>`) page that describes the features for Docker Desktop _in general_ (instead of split per platform?)

Reply:
We can combine the Mac and Windows User Manual and move it below `/desktop/`. We'll have to think about sections that are specific to each OS though. This is a separate work item. Therefore, suggest we retain the existing link for tomorrow's release and then move it to `/desktop/user-manual/` when the new page is live. 

@StefanScherer @mat007 WDYT?

Reply:
This does sound like a major refactoring indeed (for instance how to handle screenshots?).

We could split in two URLs `/go/win-desktop-pause/` and `/go/mac-desktop-pause/` if needed, the release hasn’t been cut.

Reply:
So, in general, we should try to have screenshots platform-agnostic (where possible), and (again, where possible) try to centralise things that are "common" (after all, we try to make the "Docker" experience the same, irregardless of what operating system someone is running).

But I realise that may not be possible for _everything_ (and there _will be_ some (minor) differences)

In this case, I think the feature is exactly the same, but the menu looks different on windows and mac, so we could decide to either

- not include a screenshot (looks like we don't use screenshots for all `whale -> menu-item` instructions)
- crop the screenshot to make it more "agnostic" (notice that the looks of the menu will already differ between macOS (and likely Windows) versions, and (e.g.) dark-mode/non-dark mode.
- put both screenshots side-by-side (windows/mac)

We could also work on using tabs for screenshots, and switch those based on platform and/or cookie/setting

Reply:
Yes, it's possible. I've opened a PR https://github.com/docker/pinata/pull/16697 that we could merge. 😃 
It is using `/go/win-desktop-pause/` and `/go/mac-desktop-pause/` as suggested.

Reply:
Yes, thanks Stefan!

Reply:
@usha-mandya We will use two separate /go/win-desktop-pause/ and /go/mac-desktop-pause/ links in DD 4.2.0, so you can prepare the two redirects.